packages/beacon-node/src/api/impl/node/utils.ts

```diff
@@ -44,6 +44,7 @@ function getPeerState(status: StreamStatus): routes.node.PeerState {
     case "closing":
       return "disconnecting";
     case "closed":
+      return "disconnected";
```

Comment:
why?

Reply:
Such `switch -> cases` are considered useless. One may left those mistakenly without providing their body, so either to have a case body or remove such cases. 

https://biomejs.dev/linter/rules/no-useless-switch-case/